Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM rust:1.45.0 AS builder
+
+RUN apt-get update && apt-get install -y librocksdb-dev clang
+
+WORKDIR /app
+
+COPY . .
+RUN cargo build --release
+
+FROM debian:buster-slim
```

Comment:
Why use two `FROM` instructions here? Ideally we just pull one image, then build and run the binary from there.

Reply:
it is called multi stage build : https://docs.docker.com/develop/develop-images/multistage-build/

In this way the final image will be of about 100Mb instead of 1.6GB ;-)